exercises/binary-search/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+You have stumbled upon a group of mathematicians who are also singer-songwriters.
+They have written a song for each of their favorite numbers, and, as you can imagine, they have a lot of favorite numbers.
+
+You are curious to hear one of their songs, but with so many numbers to choose from, finding the right song could take a while.
```

Comment:
On rereading this I see that it sounds like you would be happy to hear _any song_, not a particular song. I think we need to tweak the wording.

Reply:
```suggestion
You are curious to hear the song for your favourite number, but with so many songs to wad through, finding the right song could take a while.
```